src/apps/api/views/competitions.py

```diff
@@ -164,3 +189,30 @@ class CompetitionCreationTaskStatusViewSet(RetrieveModelMixin, GenericViewSet):
     queryset = CompetitionCreationTaskStatus.objects.all()
     serializer_class = CompetitionCreationTaskStatusSerializer
     lookup_field = 'dataset__key'
+
+
+class CompetitionParticipantViewSet(ModelViewSet):
+    queryset = CompetitionParticipant.objects.all()
+    serializer_class = CompetitionParticipantSerializer
+    filter_backends = (DjangoFilterBackend, SearchFilter)
+    filter_fields = ('user__username', 'user__email', 'status', 'competition')
+    search_fields = ('user__username', 'user__email',)
+
+    def get_queryset(self):
+        qs = super().get_queryset()
+        qs = qs.select_related('user').order_by('user__username')
```

Comment:
nice, was going to say from `CompetitionParticipantSerializer` make sure we do a `select_related` ;)